ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -295,7 +301,15 @@ func (rv *Clusters) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err != nil {
 		return err
 	}
-
+	rv.ManifestsConfigMapRefs = append(rv.ManifestsConfigMapRefs, ManifestsConfigMapReference{
+		Name: rv.ClusterName,
+	})
+	zapLabel, found := rv.ClusterLabels["ztp-accelerated-provisioning"]
+	if found && (zapLabel == "full" || zapLabel == "policies") {
+		rv.ManifestsConfigMapRefs = append(rv.ManifestsConfigMapRefs, ManifestsConfigMapReference{
+			Name: fmt.Sprintf("%s-aztp", rv.ClusterName),
```

Comment:
@vitus133  Where is this configMap that we are passing to AgentClusterInstall CR?

Reply:
TALM will create it from policies if managedcluster is labeled in a certain way